src/BenchmarkDotNet/Configs/ManualConfig.cs

```diff
@@ -221,6 +224,12 @@ public ManualConfig AddLogicalGroupRules(params BenchmarkLogicalGroupRule[] rule
             return this;
         }
 
+        public ManualConfig AddEventProcessor(EventProcessor eventProcessor)
```

Comment:
@adamsitnik, @caaavik-msft

> nit: it's unlikely that the users will need more than one instance

While this may be true, this makes it the only one of the `Add...` methods that doesn't accept a params array. Personally, I would prefer the API consistency.